server/api/carts.js

```diff
@@ -1,36 +1,49 @@
-const express = require('express');
+const express = require("express");
 const app = express();
-const { User, Cart } = require('../db/index');
+const { User, Cart, CartItem } = require("../db/index");
 
-app.get('/', (req, res, next) => {
-  Cart.findAll({ include: [{ model: User }] })
+app.get("/", (req, res, next) => {
+  Cart.findAll({
+    include: [{ model: CartItem, as: "CartItem" }]
+  })
     .then(carts => res.status(200).send(carts))
     .catch(err => next(err));
 });
 
-app.get('/:id', (req, res, next) => {
+app.get("/:id", (req, res, next) => {
   const { id } = req.params;
-  Cart.findOne({ where: id, include: [{ model: User }] });
+  Cart.findOne({
+    where: { id: id },
```

Comment:
deconstructed key on line 14

Reply:
This was the first case where it started throwing a "raw query prohibited" error at me... still not entirely sure what that meant. Pretty sure if I change this. It will break.

Reply:
I think I found the bug, let me branch it off and work on it and hold on to this merge for when I fix it. I'm pretty sure is a bug of my own doing.